test/e2e/app-dir/segment-cache/basic/segment-cache-basic.test.ts

```diff
@@ -0,0 +1,28 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('segment cache (basic tests)', () => {
+  const { next, isNextDev, isNextDeploy } = nextTestSetup({
+    files: __dirname,
+  })
+
+  if (isNextDev || isNextDeploy) {
+    test('ppr is disabled', () => {})
+    return
+  }
+
+  it('basic navigation', async () => {
+    const browser = await next.browser('/')
+
+    // Navigate to the test page
+    const link = await browser.elementByCss('a')
+    await link.click()
+
+    // Confirm that the content has streamed in and the URL has updated
+    // as expected
+    const nav = await browser.elementById('nav')
+    expect(await nav.innerHTML()).toMatchInlineSnapshot(
+      `"<div data-streaming-text="true"><div>Static in nav</div><div>Dynamic in nav</div></div>"`
```

Comment:
should we assert on the layout & page text as well?

Reply:
Since this is a smoke test I just picked one of them. These tests aren't meant to be comprehensive because eventually we need to run _all_ the App Router tests with the flag on, like we did with PPR.